<?xml version="1.0"?>
<launch>
    <node pkg="migrave_video_recorder" type="ros_recorder.py"
          name="migrave_video_recorder" output="screen">
        <param name="/output_directory" type="str" value='/home/qtrobot/videos/1' />
        <param name="/color_image_topic" type="str" value='/camera/color/image_raw' />
        <param name="/depth_image_topic" type="str" value='/camera/depth/image_raw' />
        <param name="/is_record_topic" type="str" value='/qt_robot_video_recording/is_record' />
        <param name="/video_type" type="str" value="mp4" />
        <param name="/video_dimensions" type="str" value="480p" />
        <param name="/frames_per_second" type="int" value="30" />
    </node>
</launch>